include/deal.II/base/synchronous_iterator.h

```diff
@@ -38,6 +41,12 @@ DEAL_II_NAMESPACE_OPEN
  * The template argument of the current class shall be of type
  * <code>std_cxx11::tuple</code> with arguments equal to the iterator types.
  *
+ * The individual iterators can be accessed using 
+ * <code>std_cxx1::get<X>(synchronous_iterator.iterators)</code> where X is the 
+ * number corresponding to the desired iterator.
+ *
+ * Additional documentation can be found under Files then File Members.
```

Comment:
I know. But look at https://www.dealii.org/developer/doxygen/deal.II/classSymmetricTensor.html -- this lists all of the related global functions on the same page, which are linked via `@relates`, e.g.
```
/**
 * Compute the determinant of a tensor or rank 2. The determinant is also
 * commonly referred to as the third invariant of rank-2 tensors.
 *
 * For a one-dimensional tensor, the determinant equals the only element and
 * is therefore equivalent to the trace.
 *
 * For greater notational simplicity, there is also a <tt>third_invariant</tt>
 * function that returns the determinant of a tensor.
 *
 * @relates SymmetricTensor
 * @author Wolfgang Bangerth, 2005
 */
template <int dim, typename Number>
inline
Number determinant (const SymmetricTensor<2,dim,Number> &t)
```
I think this is a better way.